### PR TITLE
Add the initial getstarted file

### PR DESCRIPTION
**Pull request by mitechie**
Add support for a getstarted file in the Juju Shell charm. It walks through the steps needed after deploying the charm to set up the Juju Shell.

### CODE REVIEW

**Comment by mitechie** [comment]
screenshot of the working getstarted, since this screenshot I did edit down the status output to shrink it down a bit. 

testgetstarted_-_juju_gui

File: getstarted.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    jujushell/0  waiting   allocating  0        35.196.154.94          waiting for
+    machine
+
+    Machine  State    DNS            Inst id        Series  AZ          Message
+    0        pending  35.196.154.94  juju-e62c69-0  xenial  us-east1-b  RUNNING
+
+You want to pull the `Public Address` from the status of the Juju Shell and
+set that to your DNS record.
+
+    jujushell.mydomain.com.        IN      A       35.196.154.94
```
Comment (frankban) [inline getstarted.md]: Maybe specify that jujushell.* is just an example, that it could be anything?

File: getstarted.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Once that returns the new DNS information you can set the configuration on the
+charm to trigger the Let's Encrypt SSL certificate.
+
+    $ juju config jujushell dns-name=jujushell.mydomain.com
+
+
+The final step is to configure the Juju GUI to use the Juju Shell charm at the
+DNS address you setup. Configure the Juju Shell url by typing `shift-!` and
+entering the DNS name into the field
```
Comment (frankban) [inline getstarted.md]: ... if not already there. If you visited the model in which the jujushell is deployed with the latest gui it is possible that the GUI detected the availability of a shell and set the option automatically.

**Review by frankban — APPROVED** [review]
Looks good thanks!

File: getstarted.md (hunk 1 above)
Comment (mitechie) [reply getstarted.md]: sure thing

File: getstarted.md (hunk 2 above)
Comment (mitechie) [reply getstarted.md]: updated with some wording to that effect.

**Pull request merged**